src/routes/layout.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useContext } from 'react';
+import { useLocation } from 'react-router-dom';
+import styled from 'styled-components';
+import { AppContext } from '../app-context/app-context';
+import { MainContainer } from '../styled-components/common';
+
+interface Props {
+    children: React.ReactChild | React.ReactChildren;
+}
+
+const MasterPaddingContainer = styled.div`
+    padding-bottom: 64px;
+`;
+
+const isPartOfForm = (path: string) => {
+    const leadPart = path.split('/')[2];
+    if (!leadPart) {
+        return false;
+    }
+    return leadPart === 'begrunnelse' || leadPart === 'oppsummering' || leadPart === 'kvittering';
+};
+
+const Layout = ({ children }: Props) => {
+    const location = useLocation();
+    const { klage } = useContext(AppContext);
+
+    const bgColorChooser = () => {
+        if (klage === null || !isPartOfForm(location.pathname)) {
+            return '#e7e9e9';
+        }
+        return '#fff';
+    };
+
+    const Background = styled.div<Props>`
```

Comment:
I stedet for å legge til en ekstra `div`, kan vi ikke bare gjenbruke `MainContainer`?
Dvs. sette akkurat samme bakgrunn på `MainContainer`.

Reply:
> Ifølge `styled-components` bør man definere komponenten utenfor og bruke dynamiske verdier som props.
> Som dette:
> `color: ${props => props.color};`

OK. Takk for tipset.